### PR TITLE
Add quay 3.8 tests on Hypershift 4.13

### PR DESCRIPTION
**Pull request by LiZhang19817**
Add quay 3.8 tests on Hypershift 4.13

### CODE REVIEW

File: ci-operator/config/quay/quay-tests/quay-quay-tests-master__quay38-hypershift-ocp413.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  upi-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: upi-installer
```
Comment (LiangquanLi930) [inline ci-operator/config/quay/quay-tests/quay-quay-tests-master__quay38-hypershift-ocp413.yaml:9]: add 
```
  hypershift-operator:
    name: "4.13"
    namespace: ocp
    tag: hypershift-operator
```

Comment (LiZhang19817) [reply ci-operator/config/quay/quay-tests/quay-quay-tests-master__quay38-hypershift-ocp413.yaml]: /pj-rehearse periodic-ci-quay-quay-tests-master-quay38-hypershift-ocp413-quay-e2e-tests-quay38-hypershift-ocp413

**Comment by LiZhang19817** [comment]
/pj-rehearse periodic-ci-quay-quay-tests-master-quay38-hypershift-ocp413-quay-e2e-tests-quay38-hypershift-ocp413

**Pull request closed without merging**